src/Microsoft.SqlTools.ServiceLayer/DacFx/DacFxService.cs

```diff
@@ -228,9 +228,18 @@ private void ExecuteOperation(DacFxOperation operation, DacFxParams parameters,
                 try
                 {
                     TaskMetadata metadata = TaskMetadata.Create(parameters, taskName, operation, ConnectionServiceInstance);
+                    if (operation.GetType() == typeof(ExtractOperation) || operation.GetType() == typeof(ExportOperation))
+                    {
+                        // show package file path in task viewlet instead of server and database name
+                        metadata.DatabaseName = null;
+                        metadata.ServerName = parameters.PackageFilePath;
```

Comment:
absolutely like the direction. I am feeling can we not have both pieces of info from which db to which file or is that too much for single line?

Reply:
@chlafreniere (since you opened the issue :) ) do you have any preference?